index.html

```diff
@@ -25,15 +25,18 @@
   </header>
   <main class="container">
     <section class="introduction padding_lr">
```

Comment:
- [ ] Great work on your desktop version of your portfolio! I noticed that **the desktop navbar is missing from your project.** Could you kindly consider adding it based on the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471)? Also, please remember **to hide the hamburger menu and display different navigation links there.** 🙋‍♂️

   ### Your design:
   ![image](https://github.com/wacoo/Portfolio-Project/assets/107952724/bc22a50f-22ce-4ac3-977d-f8d57977956d)

   ### Figma design:
   ![image](https://github.com/wacoo/Portfolio-Project/assets/107952724/a54f5fc4-46bd-4739-bb21-0d8179adedb3)